handbook/customer-success/README.md

```diff
@@ -0,0 +1,228 @@
+
+- Eric Shaw (fleetdm.com)
+- Zay Hanlon
+- Luke Heath
+- Mike McNeil
+
+Expected response times: during business hours, 1 hour. Outside of business hours <4 hours.
+
+The first responder on-call for Managed Cloud will take ownership of the @infrastructure-oncall alias in Slack first thing Monday morning. The previous week's on-call will provide a summary in the #g-customer-success Slack channel with an update on alarms that came up the week before, open issues with or without direct end-user impact, and other issues to keep an eye out for.
+
+Escalation of alarms will be done manually by the first responder according to the escalation contacts mentioned above. A [suspected outage issue](https://github.com/fleetdm/confidential/issues/new?assignees=&labels=%23outage%2C%23g-cx%2C%3Arelease&projects=&template=outage.md&title=Suspected+outage%3A+YYYY-MM-DD) should be created to track the escalation and determine root cause. 
+
+All infrastructure alarms (fleetdm.com and Managed Cloud) will go to #help-p1.
+
+When the current 24/7 on-call engineer is unable to meet the response time SLAs, it is their responsibility to arrange and designate a replacement who will assume the @oncall-infrastructure Slack alias.
+
+#### Runbooks
+
+Use [on-call runbooks](https://github.com/fleetdm/confidential/tree/main/infrastructure/runbooks#readme) to guide your response. Runbooks provided detailed, step-by-step instructions to quickly and effectively respond to and resolve most 24/7 on-call alerts. 
+
+## Responsibilities
+
+#### First-line response
+
+Customer Success is responsible for the first response to messages in private customer Slack channels.
+
+Customer Support Engineers are responsible for the first response to Slack messages in the [#fleet channel](https://osquery.slack.com/archives/C01DXJL16D8) of osquery Slack, and other public Slacks. 
+
+The 24/7 on-call is responsible for alarms related to fleetdm.com and Fleet Managed Cloud, as well as delivering 24/7 support for Fleet Ultimate customers.
+
+We respond within 1-hour during business hours and 4 hours outside business hours. Note that we do not need to have answers within 1 hour -- we need to at least acknowledge and collect any additional necessary information, while researching/escalating to find answers internally.
+
+> Response SLAs help us measure and guarantee the responsiveness that a customer [can expect](https://fleetdm.com/handbook/company#values) from Fleet.  But SLAs aside, when a Fleet customer has an emergency or other time-sensitive situation ongoing, it is Fleet's priority to help them find them a solution quickly.
+
+### Onboard a customer support engineer
+What do you do every day? What does the path to success look like in this role and what can you do to contribute quickly at Fleet? To onboard a customer support engineer at Fleet it's important to understand the [continued training needed](https://docs.google.com/document/d/1GB8i_VMaFxeb9ipLock9MVWGJ2RqIW8lZ5n3MLiXG4s/edit).
```

Comment:
Rather than duplicating with more words, let's just say:

"Follow the [training steps for this role](LINK)."